src/routers/handlers/company/confirm.ts

```diff
@@ -56,26 +70,26 @@ export class ConfirmCompanyHandler extends GenericHandler {
 
     const session: Session = req.session as Session;
     const companyProfile: CompanyProfile = session.data.extra_data.companyProfile;
-    if (!validationConstants.VALID_COMPANY_TYPES.includes(companyProfile.type)) {
+    if (!VALID_COMPANY_TYPES.includes(companyProfile.type)) {
       logger.info(`company confirm - invalid company type`);
-      this.viewData.invalidCompanyReason = validationConstants.INVALID_COMPANY_TYPE_REASON;
-    } else if (!validationConstants.VALID_COMPANY_STATUS.includes(companyProfile.companyStatus)) {
+      this.viewData.invalidCompanyReason = INVALID_COMPANY_TYPE_REASON;
+    } else if (!VALID_COMPANY_STATUS.includes(companyProfile.companyStatus)) {
       logger.info(`company confirm - invalid company status`);
-      this.viewData.invalidCompanyReason = validationConstants.INVALID_COMPANY_STATUS_REASON;
+      this.viewData.invalidCompanyReason = INVALID_COMPANY_STATUS_REASON;
     } else {
       try {
         logger.info(`company confirm - checking company email`);
         const companyEmail = await getCompanyEmail(companyProfile.companyNumber);
         logger.info(`company confirm - company email found: ${companyEmail}`);
-        session?.setExtraData(constants.REGISTERED_EMAIL_ADDRESS, companyEmail.registeredEmailAddress);
+        session?.setExtraData(REGISTERED_EMAIL_ADDRESS, companyEmail.registeredEmailAddress);
       } catch (e) {
         const error = e as Error;
-        if (error?.name === constants.SERVICE_UNAVAILABLE) {
+        if (error?.name === THERE_IS_A_PROBLEM_ERROR) {
           logger.info(`company confirm - oracle query service unavailable`);
-          this.viewData.invalidCompanyReason = validationConstants.INVALID_COMPANY_SERVICE_UNAVAILABLE;
+          this.viewData.invalidCompanyReason = INVALID_COMPANY_SERVICE_UNAVAILABLE;
         } else {
           logger.info(`company confirm - company email not found`);
-          this.viewData.invalidCompanyReason = validationConstants.INVALID_COMPANY_NO_EMAIL_REASON;
+          this.viewData.invalidCompanyReason = INVALID_COMPANY_NO_EMAIL_REASON;
```

Comment:
It would be better if the INVALID_COMPANY_SERVICE_UNAVAILABLE, its unlikely but with this we could end up displaying a invalid no email page to a company with a rea.